src/libraries/Microsoft.PowerFx.Core/IR/Nodes/BinaryOpNode.cs

```diff
@@ -29,7 +29,7 @@ public BinaryOpNode(IRContext irContext, BinaryOpKind op, IntermediateNode left,
 
         public override string ToString()
         {
-            return $"BinaryOp({Op}, {Left}, {Right})";
+            return $"{Op}:{IRContext.ResultType._type}({Left}, {Right})";
```

Comment:
>{IRContext.ResultType._type} [](http://example.com/codeflow?start=26&length=28)

I like the format for booleans/numbers where you have ":b" or ":n" suffix.
Here it's a prefix.
Not sure which one is best but it could be nice to be consistent

Reply:
This is postfix notation too.  `1+2` results in `AddNumbers:n(1:n, 2:n)`

Reply:
Why not `AddNumbers(1:n, 2:n):n` ?